openedx/core/djangoapps/enrollments/api.py

```diff
@@ -20,14 +20,14 @@
 DEFAULT_DATA_API = 'openedx.core.djangoapps.enrollments.data'
 
 
-def get_enrollments(user_id, include_inactive=False):
+def get_enrollments(username, include_inactive=False):
     """Retrieves all the courses a user is enrolled in.
 
     Takes a user and retrieves all relative enrollments. Includes information regarding how the user is enrolled
     in the the course.
 
     Args:
-        user_id (str): The username of the user we want to retrieve course enrollment information for.
+        username: The username of the user we want to retrieve course enrollment information for.
```

Comment:
Why are you removing the type information when you change this from `user_id` to `username`?

Reply:
@feanil Because sometimes the value is a unicode instead of a string.

Reply:
Ok.